### ISSUE TITLE
SCB: LocationPane: issues / ideas for improvement

### ISSUE THREAD

**Issue by bastiansteinert** [LivelyKernel/LivelyKernel#153]
text partially copied from a mail: 

In the file path, when the user has a specific file selected, the file name itself could be included. The user wants to copy this out and get a more-or-less complete path.

While it is nice to take off the path before /core/, it is unnerving to see nothing in that space when you are looking at /core/. One might prefer to see the entire path all the time.  Naive user don't know what is going on.  Maybe a menu item with 'copy full url' to let them see the real complete path.

The switch between [Codebase] and [local] is hidden.  New users will never know to right click to find that hidden switch!  How about a small button the brings up a menu?

The name 'local' should be changed to something more appropriate such as 'Your Directory' ... 
Note:
If the world is on a server, [local] lets you see:
lively-kernel.org/core/users/YourName/
If you are running locally, [local] lets you see:
localhost:9001/users/YourName/

Perhaps that same menu could include an item: [ copy do-it ].  This would copy the text (for example) '$world.browseCode("lively.morphic.World", "createStatusMessage", "lively.morphic.MorphAddons");'.   The purpose is to let a user paste this text into email or skype.  The user needs to tell another person where to find a specific piece of code.

In summary, an additional button specific for the LocationPane could bring up this menu:
[Go to Codebase]
[Go to Your Directory]
[Copy full URL]
[Copy path do-it]
